pkg/udevevent/monitor.go

```diff
@@ -98,12 +117,35 @@ func (m *monitor) process(fd int) error {
 		device.UdevDeviceUnref()
 		return nil
 	}
-	event := newEvent()
-	event.process(device)
-	event.send()
+	var eventType UdevEventType
+	switch device.GetAction() {
+	case libudevwrapper.UDEV_ACTION_ADD:
+		eventType = EventTypeAdd
+	case libudevwrapper.UDEV_ACTION_REMOVE:
+		eventType = EventTypeRemove
+	default:
+		eventType = EventTypeChange
```

Comment:
Should we default to change, or just ignore?

Reply:
throwing error in case it doesn't match